addons/purchase_stock/tests/test_stockvaluation.py

```diff
@@ -1136,7 +1136,7 @@ def test_average_realtime_anglo_saxon_valuation_multicurrency_same_date(self):
         inv.action_post()
 
         move_lines = inv.line_ids
-        self.assertEqual(len(move_lines), 4)
+        self.assertEqual(len(move_lines), 2)
```

Comment:
This was initially 2 but it was changed to for [In PR](https://github.com/odoo/odoo/pull/110715/files) even though it should be 2 for `average`  costing method but IMO it should be 2
[it should not create extra lines for non `standard` cost method](https://github.com/odoo/odoo/blob/master/addons/purchase_stock/models/account_invoice.py#L43)

checking in detail just to be sure